### ISSUE TITLE
Show all matching synonyms and index entries, not just one.

### ISSUE THREAD

**Issue by ecraven** [Dushistov/sdcv#80]
A dictionary can contain multiple synonym and index entries for one string, provide an option to show *all* of them, don't just pick one.

Would a PR for such a feature be acceptable?

**Comment by Dushistov** [comment]
Yes, it is actually bug if `sdcv` does not show all entries and instead of it pickup up just one.

**Comment by ecraven** [comment]
@Dushistov it seems this is already fixed, however the latest tag (0.5.3) does not contain the fix. Could you be convinced to create a new tag 0.5.4?

**Comment by Dushistov** [comment]
> Could you be convinced to create a new tag 0.5.4?

Done. If this issue fixed for you, please consider to close it.

**Comment by ecraven** [comment]
Sorry it took so long, this has been fixed for a long time, thanks for fixing it!
